### PR TITLE
AV-210932 Improve UT runtime (integrationtest)

### PR DESCRIPTION
**Pull request by aaha97**
this PR isolates changes required for https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/pull/1339

file changes for integrationtests

UT results

ok      github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tests/integrationtest       500.357s

### CODE REVIEW

File: tests/integrationtest/l4_service_test.go [hunk 1]
```diff
@@ -1796,7 +1837,7 @@ func TestLBSvcWithExtDNSAndAutoFQDNAsFlat(t *testing.T) {
 	g := gomega.NewGomegaWithT(t)
 	SetUpTestForSvcLBWithExtDNS(t)
 
-	modelName := "admin/cluster--red-ns-" + EXTDNSSVC
+	modelName := MODEL_REDNS_PREFIX + EXTDNSSVC
```
Comment (pkoshtavmware) [inline tests/integrationtest/l4_service_test.go:1840]: where is MODEl_REDNS_PREFIX defined ?
I couldn't find this in lib.go

File: tests/integrationtest/l4_crd_test.go [hunk 2]
```diff
@@ -95,13 +95,15 @@ func TestCreateDeleteL4RuleInvalidLBClass(t *testing.T) {
 	// this test checks the following scenario:
 	// adding a valid l4rule crd annotation to an invalid LBSvc should not cause the VS to come up
 	g := gomega.NewGomegaWithT(t)
-	L4RuleName := "test-l4rule"
+	L4RuleName := objNameMap.GenerateName("test-l4rule")
+	svcName := objNameMap.GenerateName(SINGLEPORTSVC)
+	modelName := MODEL_REDNS_PREFIX + svcName
```
Comment (pkoshtavmware) [inline tests/integrationtest/l4_crd_test.go:100]: where is MODEl_REDNS_PREFIX defined ?
I couldn't find this in lib.go

Comment (aaha97) [reply tests/integrationtest/l4_crd_test.go:100]: defined in l4_service_test.go

**Review by pkoshtavmware — APPROVED** [review]
LGTM

**Review by akshayhavile — APPROVED** [review]

**Pull request merged**